.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: Prepare Release
+# Prepare Release looks for a tag with a "prepare" prefix.
+# If encountered an automated changelog pull request will be opened against the repository.
+# If the tag does not contain alpha, beta, or rc test artifacts will be created for QE testing.
```

Comment:
This comment seems incorrect. I can't see how this is triggered from a tag

Reply:
yes its wrong.